### PR TITLE
Added ISSUE_TEMPLATE 

### PR DESCRIPTION
**Pull request by RahulSoni0**
## Description

Created a new issues template 

## Issue Resolved

fixed Create an issue template #15 

## Checklist

Please make sure to review the following before submitting your PR: 

- [x] I have read the [contribution guidelines](https://github.com/PEC-CSS/MovieDroid/blob/main/CONTRIBUTING.md).
- [x] I have read the code of conduct.
- [x] I have reviewed my submission in detail.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "important","updates"]
+assignees:
```
Comment (yellowHatpro) [inline .github/ISSUE_TEMPLATE/bug_report.yaml]: you can remove the assignees tag, this is optional and is set to octocat.

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
```
Comment (yellowHatpro) [inline .github/ISSUE_TEMPLATE/bug_report.yaml]: Please also remove the version part, it is not needed as of now.

**Review by yellowHatpro — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 2 above)
Comment (RahulSoni0) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: done

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (RahulSoni0) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: done, but assignees section is auto generated

**Review by yellowHatpro — APPROVED** [review]
looks good now, thanks for the work 💯

**Pull request merged**